docs/paper/dev/getting-started/project-setup.md

```diff
@@ -0,0 +1,133 @@
+---
+slug: /dev/project-setup
+---
+
+# Paper Project Setup
+
+As the Paper team primarily uses [IntelliJ IDEA](https://www.jetbrains.com/idea/), this guide will be focused on that IDE. 
+However, the steps below should be applicable to other IDEs as well, with some minor changes.
+
+
+### Creating a new project
+
+Open your IDE and select the option to create a new project. 
+In Intellij, you will get the option to select the type of project you want to create. 
+Select `Gradle - Kotlin DSL` and click `Create`.
+
+You will land into the `build.gradle.kts` file where you can add your dependencies.
+
+### Adding Paper as a dependency
+
+To add Paper as a dependency, you will need to add the Paper repository to your `build.gradle.kts` file as well as the dependency itself.
+
+```kotlin
+repositories {
+    mavenCentral()
+    maven {
+        url = uri("https://repo.papermc.io/repository/maven-public/")
```

Comment:
Could just use the shorthand `maven("https://repo.papermc.io/repository/maven-public/")` here

Reply:
sure - note that this is copied from the paper readme

Reply:
average paper doc quality smh